From 48f15470e63d746e88bcbfcc9b5a0ce3037f8b64 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:22:09 -0400
Subject: [redhat] kabi: Adding symbol __init_waitqueue_head (kernel/wait.c)

Message-id: <6ed7bb7b6788418d72a7cb2552ec4d1b854a17ea.1410510773.git.poros@redhat.com>
Patchwork-id: 95901
O-Subject: [PATCH RHEL7.1 009/233] kabi: Adding symbol __init_waitqueue_head (kernel/wait.c)
Bugzilla: 1062361 1071097
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1062361,1071097
    https://bugzilla.redhat.com/show_bug.cgi?id=1062361
    https://bugzilla.redhat.com/show_bug.cgi?id=1071097

    Description:
    ============
    This is not referenced directly, it gets included when using macro
    init_waitqueue_head. This macro is used to initialize the waitqueue head.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/__init_waitqueue_head
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/__init_waitqueue_head
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/__init_waitqueue_head

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/__init_waitqueue_head b/redhat/kabi/kabi-module/kabi_ppc64/__init_waitqueue_head
new file mode 100644
index 0000000..bbdf814
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/__init_waitqueue_head
@@ -0,0 +1,2 @@
+#1-
+0xa1c99385 __init_waitqueue_head vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/__init_waitqueue_head b/redhat/kabi/kabi-module/kabi_s390x/__init_waitqueue_head
new file mode 100644
index 0000000..7ccff1d
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/__init_waitqueue_head
@@ -0,0 +1,2 @@
+#1-
+0xac213e3a __init_waitqueue_head vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/__init_waitqueue_head b/redhat/kabi/kabi-module/kabi_x86_64/__init_waitqueue_head
new file mode 100644
index 0000000..e264195
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/__init_waitqueue_head
@@ -0,0 +1,2 @@
+#1-
+0xf432dd3d __init_waitqueue_head vmlinux EXPORT_SYMBOL
-- 
1.7.1